linkml/generators/sqltablegen.py

```diff
@@ -201,7 +213,7 @@ def strip_newlines(txt: Optional[str]) -> str:
                     #    is_pk = True  ## TODO: use unique key
                     args = []
                     if s.range in schema.classes and self.use_foreign_keys:
-                        fk = sql_name(self.get_foreign_key(s.range, sv))
+                        fk = sql_name(self.get_id_or_key(s.range, sv))
```

Comment:
Renamed this function, since it doesn't actually get foreign keys; it just retrieves the `id` or `key` slot.